Yoast/wordpress-seo
# Add back sitedesc replacevar in search appearance

## Summary

This PR can be summarized in the following changelog entry:

* Corrected a bug where the "Tagline" / "%%sitedesc%%" snippet editor variable was not selectable in the Search-Appearance settings.

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:

* Go to search appearance and verify that entering `%%sitedesc%%` gets serialized to `Tagline`, and that pressing `%` and `t` will show tagline in the dropdown.

## Quality assurance


Fixes #10308
